docs/builder/token-promotion.md

```diff
@@ -0,0 +1,72 @@
+# Free Microsoft Developer Account Promotion
+
+For a limited time, PWABuilder is offering free Microsoft Store Developer accounts (normally $19) to individual developers who have a progressive web app that meets specific requirements. With a Microsoft Sore Developer account, you can deploy your progressive web app to the Microsoft Store, extending your app's reach to millions of users on Windows.
```

Comment:
To match our other places where we say this, lets do `approximately $19 value` instead of normally $19

Reply:
Also, Lets capitalize Progressive Web App (with maybe a link to our docs PWA intro?)

Reply:
Also, Microsoft Store is mis-spelled

Reply:
Finally, lets double check that we can say `millions of users on Windows` with Beth or Nikola

Reply:
Just going to edit that line to save time, don't think its a make or break 😄